packages/utils/lib/logger.ts

```diff
@@ -41,10 +42,12 @@ if (!isCloud && !isEnterprise) {
         })
     ];
 } else {
+    const instanceId = isCloud ? ` ${nanoid(6)}` : '';
+
     formatters = [
         winston.format.printf((info) => {
             const splat = info[SPLAT] && info[SPLAT].length > 0 ? JSON.stringify(info[SPLAT]) : '';
-            return `[${info.level.toUpperCase()}]${info['service'] ? ` [${info['service']}] ` : ''}${info.message} ${splat}`;
+            return `[${info.level.toUpperCase()}]${instanceId}${info['service'] ? ` [${info['service']}] ` : ''}${info.message} ${splat}`;
```

Comment:
You have access to `RENDER_INSTANCE_ID` that does exactly what you want 
https://render.com/docs/environment-variables#all-runtimes

Reply:
It's just the hostname, which is `srv-cfifsvha6gdq1k7u46a0-99959bc5b-6mcdc` on one of our current instances. Not sure if we want to dump something like that into every log message we have vs. a 6 character identifier.